cloud_experts_tutorials/cloud-experts-deploying-application/cloud-experts-deploying-application-networking.adoc

```diff
@@ -0,0 +1,62 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="cloud-experts-deploying-application-networking"]
+= Tutorial: Networking
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: cloud-experts-deploying-application-networking
+
+toc::[]
+
+//rosaworkshop.io content metadata
+//Brought into ROSA product docs 2023-12-14
+
+This tutorial shows how the OSToy app uses intra-cluster networking to separate functions by using microservices and visualize the scaling of pods.
+
+image::deploying-networking-arch.png[OSToy Diagram]
+
+The diagram shows there are at least 2 separate pods, each with its own service. 
+
+One pod functions as the front end web application with a service and a publicly accessible route. The other pod functions as the backend microservice with a service object so that the front end pod can communicate with the microservice. This communication occurs across the pods if more than one. Because of these communication limits, this microservice is not accessible from outside this cluster or from other namespaces or projects if these are configured. The sole purpose of this microservice is to serve internal web requests and return a JSON object containing the current hostname, which is the pod's name, and a randomly generated color string. This color string is used to display a box with that color displayed in the tile titled "Intra-cluster Communication".
+
+For more information about the networking limitations, see xref:../../networking/openshift_network_security/network_policy/about-network-policy.adoc#about-network-policy[About network policy].
+
+== Intra-cluster networking
+
+You can view your networking configurations in your OSToy application.
+
+.Procedure
+. In the OSToy application, click *Networking* in the left menu. 
+. Review the networking configuration. The right tile titled "Hostname Lookup" illustrates how the service name created for a pod can be used to translate into an internal ClusterIP address.
++
+image::deploying-networking-example.png[OSToy Networking page]
+
+. Enter the name of the microservice created in the right tile ("Hostname Lookup") following the format of `<service_name>.<namespace>.svc.cluster.local`. You can find this service name in the service definition of `ostoy-microservice.yaml` by running the following command: 
```

Comment:
The IBM style guide recommends avoiding parentheticals, but I think it's fine here.